### DISCUSSION TITLE
Home assistant e Ecowizz GW2001A

### DISCUSSION THREAD

**Discussion by matteocaretta72** [indomus/forum#2948] [GENERALE - Discussioni generiche]
Ciao a tutti,
sono nuovo nel forum, sono un programmatore di PLC industriali e da un po di tempo sto entrando nel mondo di home assistant, ho un HA yellow, al quale sto integrando piano piano il PLC di casa e alcuni sensori di domotica, ultimo sensore collegato una stazione meteo Ecowizz GW2001A, tramite HACS e poi l'integrazione; funziona quasi tutto, ovvero non integra alcune entità (Pioggia e batterie), nel Log di diagnostica mi scrive "Unhandled sensor type" e non vengono integrate. Qualcuno di Voi ha idee o da spiegarmi dove posso intervenire?
Grazie in anticipo

**Comment by snakuzzo** [comment]
Se le entità sono generate da un'integrazione (che sia custom component preso da HACS o integrazione nativa), devi segnalare la cosa sul progetto GitHub dell'integrazione, verificare se ci sono issue e nel caso aprirne una e/o contribuire alla correzione creando una PR

**Comment by matteocaretta72** [comment]
Grazie mille, fatto 

Inviato da Yahoo Mail su Android 
 
  Il Sab, 2 Mar, 2024 alle 10:07, Diego ***@***.***> ha scritto:   


Se le entità sono generate da un'integrazione (che sia custom component preso da HACS o integrazione nativa), devi segnalare la cosa sul progetto GitHub dell'integrazione, verificare se ci sono issue e nel caso aprirne una e/o contribuire alla correzione creando una PR
